tests/test_watson.py

```diff
@@ -794,6 +794,26 @@ def test_report(watson):
     with pytest.raises(WatsonError):
         watson.report(arrow.now(), arrow.now(), tags=["A"], ignore_tags=["A"])
 
+    watson.start('foobaz')
+
+    report = watson.report(
+        arrow.now(), arrow.now(), current=True, projects=['foobaz']
+    )
+    assert len(report['projects']) == 1
+    assert watson.frames['id'].count('current') == 0
+
+    report = watson.report(
+        arrow.now(), arrow.now(), current=False, projects=['foobaz']
+    )
+    assert len(report['projects']) == 0
```

Comment:
You might want to test that the `foobaz` project is included in the report, not only that one project is included (even if we know that it has to be it :relaxed:).